generators/Exercises/PascalsTriangle.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Generators.Input;
+using Generators.Output;
+using Newtonsoft.Json.Linq;
+
+namespace Generators.Exercises
+{
+    public class PascalsTriangle : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var dataCases in canonicalData.Cases)
+            {
```

Comment:
I would change dataCases to dataCase (most use canonicalDataCase). Since the variable is a single case.